EnviroSense.Web/ViewModels/ApiKeys/DetailsDeviceApiKeyViewModel.cs

```diff
@@ -1,12 +1,10 @@
-﻿namespace EnviroSense.Web.ViewModels.DeviceApiKeys;
+﻿namespace EnviroSense.Web.ViewModels.ApiKeys;
 
 public class DetailsDeviceApiKeyViewModel
 {
-    public string? DeviceName { get; set; }
-    public Guid DeviceId { get; set; }
-
+    public string? KeyName { get; set; }
     public Guid Id { get; set; }
-    public string? Name { get; set; }
+    public string? Owner { get; set; }
```

Comment:
Do we really need this field to display? because owner is always the same, "you".

we will never create api keys for other accounts